src/subscriptions/SubscriptionsScreen.js

```diff
@@ -17,6 +17,17 @@ class SubscriptionsScreen extends React.Component {
     subscriptions: [],
   };
 
+  state: {
+    filter: string,
+  };
+
+  constructor() {
+    super();
+    this.state = {
+      filter: '',
```

Comment:
We "normally"  initialize in constructor but as a class variable we define types we expect